nautobot/core/ui/object_detail.py

```diff
@@ -399,8 +399,6 @@ def __init__(
         table_class,
         table_filter=None,
         table_attribute=None,
-        select_related_fields=None,
-        prefetch_related_fields=None,
```

Comment:
To be clear, I think there's still cases where specifying these may be necessary (automatic optimization can only be so smart), so I wouldn't remove the option entirely.